tests/inputFields.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { test, expect } from '@playwright/test';
+
+
+test.describe('Input fields', () => {  
+test.beforeEach( async({page}) => {
+  await page.goto('/')
+  await page.getByRole('link', { name: 'Pet Types' }).click();
+  await page.getByRole('heading', { name: 'Pet Types' }).click()
+})
+
+test('Update pet type', async ({page}) => {
+const tableRow = page.locator('tbody tr')
```

Comment:
This name of the constant is slightly incorrect, because it returns all table rows instead of a single row.
Also, to address your concern about using the indexes for edit buttons, you can here specify which row you are working with, so later, you will not need to provide index, because every time the correct row will be selected.
```
const firstTableRow = page.locator('tbody tr').first()
await firstTableRow.getByRole('button', {name: 'Edit'}).click()
```